packages/sample-controllers/src/index.ts

```diff
@@ -1,3 +1,10 @@
+export type {
+  SampleGasPricesServiceActions,
+  SampleGasPricesServiceEvents,
+  SampleGasPricesServiceMessenger,
+} from './sample-gas-prices-service/sample-gas-prices-service';
+export type { SampleGasPricesServiceFetchGasPricesAction } from './sample-gas-prices-service/sample-gas-prices-service-method-action-types';
```

Comment:
These import paths are a bit annoying, now that I look at it. I'm curious whether flattening them is a good idea.